test/cluster/object_store/conftest.py

```diff
@@ -20,16 +19,10 @@ def pytest_addoption(parser):
     s3_options.addoption('--aws-secret-key')
     s3_options.addoption('--aws-region')
     s3_options.addoption('--s3-server-bucket')
-
-    parser.addoption('--manager-api', action='store', required=True,
-                     help='Manager unix socket path')
-    parser.addoption("--artifacts_dir_url", action='store', type=str, default=None, dest="artifacts_dir_url",
-                     help="Provide the URL to artifacts directory to generate the link to failed tests directory "
-                          "with logs")
-    parser.addoption('--auth_username', action='store', default=None,
-                        help='username for authentication')
-    parser.addoption('--auth_password', action='store', default=None,
-                        help='password for authentication')
+    parser.addoption("--input", action="store", default="",
+                     help="Input file")
+    parser.addoption("--output", action="store", default="",
+                     help="Output file")
```

Comment:
I don't understand how you changed four options into two **different** options :-) What does this mean? Are you sure this change is valid? @tchaikov or @xemul can you please review this change?

Reply:
i'm sure. now test has exactly same options as before. 

before option was sum of options In this function plus option test.pylib.cql_repl.conftest.pytest_addoption and test.conftet.pytest_addoption

now it is sum of pytest_addoption funcluons in test/conftest, test/cluster/conftest, test/cluster/object_store/conftest

 four deleted options and most options from test.pylib.cql_repl.conftest.pytest_addoption come from   test.cluster.conftest.pytest_addoption I just added the rest from test.pylib.cql_repl.conftest.pytest_addoption